stored_requests/events/events.go

```diff
@@ -60,10 +60,10 @@ func (e *EventListener) Stop() {
 func (e *EventListener) Listen(cache stored_requests.Cache, events EventProducer) {
 	for {
 		select {
-		case update := <-events.Updates():
-			cache.Update(context.Background(), update.Requests, update.Imps)
-			if e.onUpdate != nil {
-				e.onUpdate()
+		case save := <-events.Saves():
+			cache.Save(context.Background(), save.Requests, save.Imps)
+			if e.onSave != nil {
```

Comment:
Should we remove the Update method from the Cache altogether if it's not being used or should we expose Save events in addition to Update events and have producers decide which to use?

Reply:
yeah, I'm fine with that. Dead code is almost always more confusing than useful. "You ain't gonna need it" and all that.

Reply:
I chatted with Hans offline... we agreed that deleting was fine.